<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsOWSSourceSelectBase</class>
 <widget class="QDialog" name="QgsOWSSourceSelectBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Layer(s) from a Server</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../.designer/backup</normaloff>../../../../.designer/backup</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="mDialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="mStatusLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ready</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="mTabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mLayersTab">
      <attribute name="title">
       <string>Layers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="mConnectionsComboBox"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="mConnectButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>C&amp;onnect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mNewButton">
           <property name="text">
            <string>&amp;New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mEditButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mDeleteButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>82</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="mLoadButton">
           <property name="toolTip">
            <string>Load connections from file</string>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mSaveButton">
           <property name="toolTip">
            <string>Save connections to file</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mAddDefaultButton">
           <property name="statusTip">
            <string>Adds a few example WMS servers</string>
           </property>
           <property name="whatsThis">
            <string comment="Adds several example WMS servers to the list"/>
           </property>
           <property name="text">
            <string>Add default servers</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="mLayersTreeWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Abstract</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="mTimeLabel">
           <property name="text">
            <string>Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mTimeComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="mCRSGroupBox">
         <property name="title">
          <string>Coordinate Reference System</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="mSelectedCRSLabel">
            <property name="text">
             <string>Coordinate Reference System</string>
            </property>
            <property name="buddy">
             <cstring>mChangeCRSButton</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mChangeCRSButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Change ...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mImageFormatsGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>64</height>
          </size>
         </property>
         <property name="title">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mWMSGroupBox">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="mLayerNameLabel">
            <property name="text">
             <string>Layer name</string>
            </property>
            <property name="buddy">
             <cstring>mLayerNameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="mLayerNameLineEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="mTileWidthLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="mTileSizeLabel">
            <property name="text">
             <string>Tile size</string>
            </property>
            <property name="buddy">
             <cstring>mTileWidthLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="mTileHeightLineEdit"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="mFeatureCountLabel">
            <property name="text">
             <string>Feature limit for GetFeatureInfo</string>
            </property>
            <property name="buddy">
             <cstring>mFeatureCountLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="mFeatureCountLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="mCacheLabel">
           <property name="text">
            <string>Cache</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mCacheComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Cache preference

Always cache: load from cache, even if it expired

Prefer cache: load from cache if available, otherwise load from network. Note that this can return possibly stale (but not expired) items from cache

Prefer network: default value; load from the network if the cached entry is older than the network entry

Always network: always load from network and do not check if the cache has a valid entry (similar to the &quot;Reload&quot; feature in browsers)
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mLayerOrderTab">
      <attribute name="title">
       <string>Layer Order</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="mLayerUpButton">
         <property name="toolTip">
          <string>Move selected layer UP</string>
         </property>
         <property name="text">
          <string>Up</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="mLayerDownButton">
         <property name="toolTip">
          <string>Move selected layer DOWN</string>
         </property>
         <property name="text">
          <string>Down</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>391</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTreeWidget" name="mLayerOrderTreeWidget">
         <property name="columnCount">
          <number>2</number>
         </property>
         <column>
          <property name="text">
           <string>Layer</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Style</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mTilesetsTab">
      <attribute name="title">
       <string>Tilesets</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="mTilesetsTableWidget">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Layers</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Styles</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Size</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Format</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>CRS</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mSearchTab">
      <attribute name="title">
       <string>Server Search</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="mSearchTermLineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="mSearchButton">
         <property name="text">
          <string>Search</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTableWidget" name="mSearchTableWidget">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>URL</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="mSearchAddButton">
         <property name="text">
          <string>Add selected row to WMS list</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mTabWidget</tabstop>
  <tabstop>mConnectionsComboBox</tabstop>
  <tabstop>mLayersTreeWidget</tabstop>
  <tabstop>mLayerNameLineEdit</tabstop>
  <tabstop>mLayerUpButton</tabstop>
  <tabstop>mLayerDownButton</tabstop>
  <tabstop>mLayerOrderTreeWidget</tabstop>
  <tabstop>mTilesetsTableWidget</tabstop>
  <tabstop>mSearchTermLineEdit</tabstop>
  <tabstop>mSearchButton</tabstop>
  <tabstop>mSearchTableWidget</tabstop>
  <tabstop>mSearchAddButton</tabstop>
  <tabstop>mDialogButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mDialogButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsOWSSourceSelectBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
